frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1070,6 +1081,51 @@ pub mod pallet {
 			<QueuedSolution<T>>::put(solution);
 			Ok(())
 		}
+
+		/// Submit a signed solution during the emergency phase. It will go through the
+		/// `feasibility_check` right away.
+		///
+		/// The dispatch origin for this call must be __signed__.
+		///
+		/// The deposit that is reserved might be rewarded or slashed based on the outcome.
+		#[pallet::weight(T::DbWeight::get().reads_writes(1, 1))]
```

Comment:
@Szegoo this weight is not correct -- you should also account for the feasibility check, which has its weight individually calculated.